recipes/glog/all/conanfile.py

```diff
@@ -37,31 +42,36 @@ def _source_subfolder(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if self.settings.os not in ["Linux", "FreeBSD"]:
+            del self.options.with_unwind
 
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
         if self.options.with_gflags:
             self.options["gflags"].shared = self.options.shared
+            self.options["libunwind"].shared = self.options.shared
```

Comment:
```suggestion
            self.options["libunwind"].shared = self.options.shared
```

are you sure it's needed?

Reply:
Well, at least it has the option. I'm not sure to be honest..

Reply:
I removed it